Neos.Flow/Classes/ObjectManagement/DependencyInjection/PropertyInjectionTrait.php

```diff
@@ -27,14 +27,18 @@ trait PropertyInjectionTrait
      * @param callable $lazyInjectionResolver
      * @return void
      */
-    private function Flow_Proxy_LazyPropertyInjection($propertyObjectName, $propertyClassName, $propertyName, $setterArgumentHash, callable $lazyInjectionResolver)
+    private function Flow_Proxy_LazyPropertyInjection(string $propertyObjectName, string $propertyClassName, string $propertyName, string $setterArgumentHash, callable $lazyInjectionResolver)
     {
         $injection_reference = &$this->$propertyName;
         $this->$propertyName = \Neos\Flow\Core\Bootstrap::$staticObjectManager->getInstance($propertyObjectName);
         if ($this->$propertyName === null) {
             $this->$propertyName = \Neos\Flow\Core\Bootstrap::$staticObjectManager->getLazyDependencyByHash($setterArgumentHash, $injection_reference);
             if ($this->$propertyName === null) {
                 $this->$propertyName = \Neos\Flow\Core\Bootstrap::$staticObjectManager->createLazyDependency($setterArgumentHash, $injection_reference, $propertyClassName, $lazyInjectionResolver);
+                if ($this->$propertyName === null) {
+                    // Fall back to eager injection, if lazy property injection was not possible
+                    $this->$propertyName = \Neos\Flow\Core\Bootstrap::$staticObjectManager->get($propertyObjectName);
+                }
             }
         }
```

Comment:
hmm it might add readability what do you say ^^

```suggestion
    $this->$propertyName = \Neos\Flow\Core\Bootstrap::$staticObjectManager->getInstance($propertyObjectName)
        ?? \Neos\Flow\Core\Bootstrap::$staticObjectManager->getLazyDependencyByHash($setterArgumentHash, $injection_reference)
        ?? \Neos\Flow\Core\Bootstrap::$staticObjectManager->createLazyDependency($setterArgumentHash, $injection_reference, $propertyClassName, $lazyInjectionResolver)
        // Fall back to eager injection, if lazy property injection was not possible
        ?? \Neos\Flow\Core\Bootstrap::$staticObjectManager->get($propertyObjectName);
```

Reply:
Please, let's postpone such changes to a follow-up. There is so much code which can be style-improved, let's dedicate a new PR to those.